spec/models/institution_spec.rb

```diff
@@ -134,10 +134,10 @@
       end
     end
 
-    it "nil or empty returns all institutions" do
+    it "nil or empty returns up to max returned institutions" do
       [nil, "", "      "].each do |arg|
         inst = Institution.autocomplete(arg)
-        expect(inst.size).to eql(Institution.all.count)
+        expect(inst.size).to eql([Institution::AUTOCOMPLETE_MAX, Institution.all.count].min)
```

Comment:
The mock Institution database had 41 entries...close enough that if we decided to tweak the autocomplete_max upwards, it would be less than the max and require updating the spec.